webpack/InsightsCloudSync/InsightsCloudSyncActions.js

```diff
@@ -0,0 +1,10 @@
+import API from 'foremanReact/API';
+import { insightsCloudUrl } from './InsightsCloudSyncHelpers';
+
+export const syncInsights = () => dispatch => {
+  API.post(insightsCloudUrl('tasks'));
```

Comment:
I think that right now because there's no reducer that needs to act upon this action
this code can be moved to the button component to an event handler:
```js
...
const handleClick = () => {
  API.post(insightsCloudUrl('tasks'))
}

...
return <Button onClick={handleClick} />
```